lib/bugcrowd/client.rb

```diff
@@ -27,7 +29,9 @@ def inspect
 
     def get(path, options = {})
       options.merge!(path: path)
-      connection.get(options)
+      resp = connection.get(options)
+      raise "Access Denied" if resp[:status] == 401
```

Comment:
Since we're using [Excon](https://github.com/excon/excon), we can add `expects: 200` and a built-in error will be raised for us on non-200 responses, in this case it'd be `Excon::Error::Unauthorized`. 💥 

So instead of:

```ruby
options.merge!(path: path)
connection.get(options)
```

We could use:

```ruby
options.merge!(path: path, expects: 200)
connection.get(options)
```